### PR TITLE
Fix file locations and references

### PR DESCRIPTION
**Pull request by levithomason**
Looks like something went awry with the merge of the build or something.  `lib/cli.js` currently fails to run.  This PR:

1. removes the unused `bin` directory
1. calls `shx` from `cli.js`
1. moves `parseArgs.js` into `src` from project root
1. fixes `no-else-return` lint error

@nfischer, here you are :)

image

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -62,12 +62,10 @@ function shx(argv) {
...
 
-  if (failed) {
-    return EXIT_CODES.CMD_FAILED;
-  } else {
-    return EXIT_CODES.SUCCESS;
-  }
+  return failed ? EXIT_CODES.CMD_FAILED : EXIT_CODES.SUCCESS;
 }
 
+process.exit(shx(process.argv));
```
Comment (ariporad) [inline src/cli.js]: We had this in a different file to make it easier to test. (argv -> exit code).

Comment (levithomason) [inline src/cli.js]: Makes sense, I'll separate the functions into individual files and put this call/exit in the cli.js.

**Comment by levithomason** [comment]
I've separated each function into it's own module.  `cli.js` now just runs `shx` with argv.

This way each file is dedicated to a single module.  Will make testing and maintenance much easier.

**Comment by nfischer** [comment]
LGTM

**Comment by levithomason** [comment]
:shipit:

**Comment by nfischer** [comment]
Reviewing this now. I'll merge if it fixes things.

@levithomason would you be interested in coming on board the shx sub-team? We usually like to do our work in PRs and have one other LGTM before merging. The main difference would be that you'd be able to review PRs to shx, which would help get this project on the road sooner.

**Comment by ariporad** [comment]
@levithomason: Agreed. We'd love to have you.

**Comment by levithomason** [comment]
Glad you asked, I'd love to help get this project off the ground.  Let's do it.

**Comment by ariporad** [comment]
@levithomason: Awesome! Invite send!

**Pull request merged**